Bring back the Exclusive Flag (#2781)
quicwg/base-drafts: draft-ietf-quic-http.md

orphan placeholder; pushes are dependent on the client request on which the
PUSH_PROMISE frame was sent.

> Requests may override the default initial values by sending a PRIORITY frame

Reviewer: Including it at stream creation is overriding the default priority.  I think I'd just call this "update" and say that everything has default priority when it's first created.

```suggestion
Priorities can be updated by sending a PRIORITY frame
```
\+ reflow

  PT (Prioritized Element Type):

>   : A two-bit field indicating the type of element being prioritized (see

Reviewer: We probably want to retain the "MUST NOT be set to `11` here, even if we call it out as an error later.

Author: Makes sense. Done in 0fd27179.

| 00      | Request stream   | Stream ID                       |
| 01      | Push stream      | Push ID                         |

> | 10      | Placeholder      | Placeholder ID                  |

Reviewer: Since the value of `11` was the only difference between these, we could re-combine them to a single table and simply say that prioritizing Root is invalid.  (Which you already have on line 1213.)

Author: That's a nice simplification! Done in 5cca97b.

ensure that both sides have a consistent view of the stream dependency tree.
HTTP/2 specifies priority assignments in PRIORITY frames and (optionally) in
HEADERS frames. To achieve in-order delivery of priority changes in HTTP/3,

> PRIORITY frames are sent or on the control stream.  HTTP/3 permits the

Reviewer: ```suggestion
PRIORITY frames are sent on the control stream.  HTTP/3 permits the
```
